docs/preflight/README.md

```diff
@@ -112,7 +112,14 @@ Verify installation with `kubectl tvk-preflight --help`
 ##### Windows
 NOT SUPPORTED
 
-
+**Note for Dark Site Installation**
+
+- For using the `local-registry` flag, it is mandatory to have `busybox` and `dnsutils:1.3` images to be there in the private registry.
+    > Steps for pushing images to local registry
+    - Pull the images (dnsutils:1.3 & busybox) to local machine.
+    - use the following command to push it to the local registry
+    - `docker push <local registry/image>` 
+    - Example: `docker push localhost:5000/busybox`
```

Comment:
Don’t we care about the other mage tag?

Reply:
Had specifically mentioned image tag for dnsutils because we are using a specific one (1.3), but there wasn't any for busybox. we are using latest. Will still update the readme though to make it clear.